ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -150,38 +148,6 @@ private void displayWelcomeMessage() {
     terminal.flush();
   }
 
-  public void runNonInteractively(final String input) throws Exception {
```

Comment:
Was only used from tests, so removing to avoid confusion.

Reply:
As the comment above says, this was only called from tests. Search for this function in this PR and you will see I haven't removed any calls to this function.  I've removed it to avoid confusion as people are raising PRs to change it, but it is not called from Production code.